src/linker/utilities/slurm_utils.py

```diff
@@ -11,6 +11,11 @@
 from linker.configuration import Config
 
 
+def is_on_slurm() -> bool:
+    """Returns True if the current environment is a SLURM cluster."""
+    return "SLURM_ROOT" in os.environ
```

Comment:
Oh, right, in my unmerged branch for Jenkins a better check is `return shutil.which("sbatch") is not None`